vue-app/src/api/core.ts

```diff
@@ -9,7 +10,7 @@ export const provider = new ethers.providers.StaticJsonRpcProvider(
   process.env.VUE_APP_ETHEREUM_API_URL
 )
 export const blockExplorer =
-  process.env.VUE_APP_BLOCK_EXPLORER || 'https://etherscan.io'
+  CHAIN_INFO[Number(process.env.VUE_APP_ETHEREUM_API_CHAINID)].explorer
```

Comment:
Might be nice to have a check here to make sure a valid `VUE_APP_ETHEREUM_API_CHAINID` is used, similar to how we check against registry types, e.g. https://github.com/ethereum/clrfund/blob/develop/vue-app/src/api/core.ts#L35